src/keywords.cpp

```diff
@@ -309,16 +309,16 @@ void init_keywords()
 
 static int kw_compare(const void *p1, const void *p2)
 {
-   const chunk_tag_t *t1 = (const chunk_tag_t *)p1;
-   const chunk_tag_t *t2 = (const chunk_tag_t *)p2;
+   const chunk_tag_t *t1 = reinterpret_cast<const chunk_tag_t *>(p1);
```

Comment:
Yes you are right, I was a bit concerned about the amount of reinterpret_cast but didn't check if they where actually needed. 
While making the changes I looked closer upon MD5.cpp but couldn't really decide which cast are really needed. I am therefore going to remove all changes made in this file.